cmd/zpool/os/freebsd/zpool_vdev_os.c

```diff
@@ -101,3 +101,17 @@ check_sector_size_database(char *path, int *sector_size)
 {
 	return (0);
 }
+
+void
+after_zpool_upgrade(zpool_handle_t *zhp)
+{
+	char bootfs[ZPOOL_MAXPROPLEN];
```

Comment:
Just a final style nit: blank line after variable declarations at the top of a function